manifests/complete-postgres-manifest.yaml

```diff
@@ -12,8 +12,9 @@ spec:
     zalando:
     - superuser
     - createdb
-  useLoadBalancer: true
-  allowedSourceRanges: #Load balancer source ranges
+  # enableMasterLoadBalancer: true
```

Comment:
I still prefer to have these options here uncommented with value 'false', since it's a default - what do you think?

Reply:
> have these options here uncommented with value 'false', since it's a default

These are two different behaviors. Not specifying the LB option in the Postgres manifest means "use relevant operator settings" . Setting the LB option to `false`  explicitly says "no load balancer please"